modules/op-resolver-cluster-tasks-ref.adoc

```diff
@@ -0,0 +1,693 @@
+|`PUSH_EXTRA_ARGS` |Extra parameters for the `push` command when pushing the image. |`string` |
+|`SKIP_PUSH` |Skip pushing the image to the container registry. |`string` |`false`
+|`TLS_VERIFY` |The TLS verification flag, normally `true`. |`string` |`true`
+|`VERBOSE` |Turn on verbose logging, all commands executed are added to the log. |`string` |`false`
+|===
+
+.Supported workspaces for the `buildah` task
+[options="header"]
+|===
+| Workspace | Description
+|`source` |Container build context, usually the application source code that includes a `Dockerfile` or `Containerfile` file.
+|`dockerconfig` |An optional workspace for providing a `.docker/config.json` file that Buildah uses to access the container registry. Place the file at the root of the workspace with the name `config.json` or `.dockerconfigjson`.
+|`rhel-entitlement` |An optional workspace for providing the entitlement keys that Buildah uses to access a {op-system-base-full} subscription. The mounted workspace must contains the `entitlement.pem` and `entitlement-key.pem` files.
+|===
+
+.Results that the `buildah` task returns
+[options="header"]
+|===
+| Result | Type | Description
+|`IMAGE_URL` |`string` |The fully qualified name of the image that was built.
+|`IMAGE_DIGEST` |`string` |Digest of the image that was built.
+|===
+
+.Changes from the `buildah` `ClusterTask`
+
+* The `VERBOSE` parameter was added.
+* The `BUILDER_IMAGE` parameter was removed.
+
+[discrete]
+[id="op-taskref-git-cli_{context}"]
+== git-cli
+
+The `git-cli` task runs the `git` command line utility. You can pass the full Git command or several commands to run using the `GIT_SCRIPT` parameter. If the commands need authentication to a Git repository, for example in order to complete a push, you must supply the authentication credentials.
+
+.Example usage of the `git-cli` task
+[source,yaml,subs="attributes+"]
+----
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: update-repo
+spec:
+# ...
+  tasks:
+# ...
+  - name: push-to-repo
+    taskRef:
+      resolver: cluster
+      params:
+      - name: kind
+        value: task
+      - name: name
+        value: git-cli
+      - name: namespace
+        value: openshift-pipelines
+    params:
+    - name: GIT_SCRIPT
+      value: "git push"
+    - name: GIT_USER_NAME
+      value: "Example Developer"
+    - name: GIT_USER_EMAIL
+      value: "developer@example.com"
+    workspaces:
+    - name: ssh-directory
+      workspace: ssh-workspace # <1>
+    - name: source
+      workspace: shared-workspace
+# ...
+----
+<1> In this example, `ssh-workspace` must contain the contents of the `.ssh` directory with a valid key for authorization to the Git repository.
+
+.Supported parameters for the `git-cli` task
+[options="header"]
+|===
+| Parameter | Description | Type | Default value
+|`CRT_FILENAME` |Certificate Authority (CA) bundle file name in the `ssl-ca-directory`
```

Comment:
```suggestion
|`CRT_FILENAME` |Certificate authority (CA) bundle file name in the `ssl-ca-directory`
```
Also nit, but you could bump a few of these that were hard wrapped to another line back on the same line